Explorer/Assets/DCL/Multiplayer/Connections/Rooms/Interior/IInterior.cs

```diff
@@ -0,0 +1,20 @@
+using DCL.Utilities.Extensions;
+
+namespace DCL.Multiplayer.Connections.Rooms.Interior
+{
+    public interface IInterior<T>
```

Comment:
can you add summary for this interface, please. To broad name (like `IContext` 😅)

Reply:
Sure, originally I took this naming from Rust, https://doc.rust-lang.org/book/ch15-05-interior-mutability.html